frontend/awx/resources/templates/WorkflowVisualizer/hooks/useWorkflowVisualizerToolbarActions.tsx

```diff
@@ -43,6 +43,11 @@ export function useWorkflowVisualizerToolbarActions() {
         </ToolbarItem>
       </ToolbarGroup>
       <ToolbarGroup align={{ default: 'alignRight' }}>
+        <ToolbarItem style={{ paddingTop: '5px' }}>
```

Comment:
Did we add this style to vertically center the ToolbarItem?

We can remove the custom padding style and add `alignSelf="center"` to the ToolbarItem to vertically center it. 
https://www.patternfly.org/components/toolbar#toolbaritem